packages/swatch/swatch-group.md

```diff
@@ -217,6 +251,65 @@ When available, the value of `selected` will be constructed from the `value` att
 <div>Selected: [ "color-2", "color-1", "color-3" ]</div>
 ```
 
-## Swatch modifying attributes
+### Swatch modifying attributes
+
+An `<sp-swatch-group>` element can be modified by the following attributes/properties to customize its delivery as desired for your use case: `border`, `density`, `rounding`, `shape`, and `size`. Use these in concert with each other for a variety of final visual deliveries. Applying a value for one of these attributes/properties to an `<sp-swatch-group>` element will have it forward the value to all of the `<sp-swatch>` elements that are a direct child of the group, overriding any value that may be applied directly to those children.
+
+Once applied to an `<sp-swatch-group>` element, the value of the `border`, `rounding`, `shape`, and `size` attributes/properties cannot be overridden on the children `<sp-swatch>` elements.
+
+<sp-tabs selected="shape" auto label=" Attribute Options">
+<sp-tab value="shape">Shape</sp-tab>
+<sp-tab-panel value="shape">
+
+```html
+<sp-swatch-group shape="rectangle">
+    <sp-swatch color="var(--spectrum-blue-500)"></sp-swatch>
+    <sp-swatch color="var(--spectrum-indigo-500)"></sp-swatch>
+    <sp-swatch color="var(--spectrum-purple-500)"></sp-swatch>
+</sp-swatch-group>
+```
+
+</sp-tab-panel>
+<sp-tab value="rounding">Rounding</sp-tab>
```

Comment:
Do you think it is useful to show the `rounding="none"` option? According to [the design guidelines](https://spectrum.adobe.com/page/swatch-group/#Corner-rounding-in-swatch-groups), that's actually the preferred/default corner rounding for swatch groups. We haven't shown that anywhere in the docs, however.

Along the same lines, do you think it would be useful to show the `border="none"` option? I was just thinking we should show examples of all of the available values for those attributes, since it's not super clear just from the API table.